src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -27,6 +27,10 @@
     public static final Block blockSolidSubstitution = new BlockSolidSubstitution();
     public static final Block blockHutField          = new BlockHutField();
     public static final Block blockHutGuardTower     = new BlockHutGuardTower();
+    public static final Block blockHutWareHouse      = new BlockHutWareHouse();
+    public static final Block blockConstructionTape  = new BlockConstructionTape();
+    public static final Block blockConstructionTapeCorner  = new BlockConstructionTapeCorner();
+
 
     // Deactivated for now
     // public static final Block blockBarrel        = new BlockBarrel();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)